src/day8/re_frame/trace.cljs

```diff
@@ -176,10 +178,12 @@
 
 (defn render-trace-panel []
   (let [filter-input     (r/atom "")
-        filter-items     (r/atom [])
+        filter-items     ((fnil #(r/atom %) [])
+                          (localstorage/get! "filter-items"))
         filter-type      (r/atom :contains)
         input-error      (r/atom false)]
     (fn []
+      (localstorage/set! "filter-items" (str @filter-items))
```

Comment:
Every time the trace panel renders, it is going to store the filter items to local storage. Is that what you intended?